Documentation/branch_management.md

```diff
@@ -0,0 +1,25 @@
+## Branch Managemnt
+
+### Guide
+
+- New development occurs on the master branch
+- Master branch should always have a green build!
+- Backwards-compatible bug fixes should target the master branch and ported to stable
+- Once the master branch is ready for release, it will be tagged and become the new stable branch.
+
+The etcd team adopts a rolling release model and support one stable version of etcd going forward.
+
+### Master branch
+
+The master branch is our development branch. It is where all the new features go into first.
+
+If you want to try new features, pull the master branch and play on it. But the branch is not really stable because new features may introduce bugs.
+
+Before the release of the next stable version, feature PRs will be frozen. We will focus on the testing, bug-fix and documentation for one to two weeks.
+
+### Stable branches
+
+Current stable branches are `release-0.4` and `release-2.0`.
```

Comment:
can we remove this line? Or we need to maintain it. I do not think it is necessary.